### PR TITLE
Fix a Starting Crash Bug and Code Cleanup

### PR DESCRIPTION
**Pull request by Nought57**
Fix a Starting Crash Bug.
Some Code Cleanup.

### CODE REVIEW

File: src/pocketmine/command/defaults/GamemodeCommand.php [hunk 1]
```diff
@@ -79,8 +79,15 @@ public function execute(CommandSender $sender, $currentAlias, array $args){
...
-				$target->sendMessage(new TranslationContainer("gameMode.changed"));
-				Command::broadcastCommandMessage($sender, new TranslationContainer("commands.gamemode.success.other", ['', $target->getName(), Server::getGamemodeString($gameMode)]));
+        
+			$target->sendMessage(new TranslationContainer("gameMode.changed"));
+				Command::broadcastCommandMessage($sender, new TranslationContainer("commands.gamemode.success.other", [$target->getName(), Server::getGamemodeString($gameMode)]));
+			}
+		}
+		return true;
+	}
+}
```
Comment (fuyutsuki) [inline src/pocketmine/command/defaults/GamemodeCommand.php]: please remove line 85~90

**Comment by fuyutsuki** [comment]
Travis build is failed, but seems to nice👍

**Comment by AppleDevelops** [comment]
Server.php has a syntax error on gamemode text please fix that and I'll merge it. Thanks for fixing some of my mistakes

**Comment by Nought57** [comment]
Missing upload!!
This files not test passing!!

**Comment by Nought57** [comment]
Fix a GamemodeCommand.php Bug.
Server.php uodate commit now.
Please wait until correspondence.

**Comment by Nought57** [comment]
This error is GamemodeCommand.php Bug.
Travis CI : Errors parsing src/pocketmine/command/defaults/GamemodeCommand.php
Fix now.

**Comment by Nought57** [comment]
Problem : conflicts that must be resolved.

**Comment by Nought57** [comment]
@AppleDevelops Conflicts Resolved and Test Passing.

**Comment by Nought57** [comment]
@fuyutsuki Your comments encourages me. Thank you very much.

**Pull request merged**